config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=23.5
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=23.5.0.0
+VERSION_LONG=23.5.0.2
```

Comment:
Notice the non-unary jump because #21829 hasn't been merged. So, technically, this PR merges both 23.5.0.1 and 23.5.0.2